src/ts/components/Container.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {Link} from 'react-router';
+import AppBar from 'components/AppBar';
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import Search from 'components/Search';
+
+export default function Container(props) {
+  return (
+     <div>
+      <AppBar />
+      <Header />
+      <Search />
```

Comment:
I don't know all views at the moment, I would leave this for now (I saw that one detail view has `AppBar` without `Header` so I assumed that they're not always together).